tests/cucumber-reportportal-formatter.spec.js

```diff
@@ -226,23 +647,170 @@ describe('cucumber-reportportal-formatter', () => {
       formatter.onTestStepStartedEvent(testStepStarted);
     });
 
-    it('finishTestItem should be called, clean storage', () => {
-      const spyFinishTestItem = jest.spyOn(formatter.reportportal, 'finishTestItem');
+    describe('FAILED status', () => {
```

Comment:
```suggestion
    describe('finishsed with failed status', () => {
```
I suggest to keep the same letter case for the same stuff (like statuses in this case).